tools/vscode/src/bin/gen_launch_json.rs

```diff
@@ -0,0 +1,159 @@
+use std::fs;
+use std::path::Path;
+
+use anyhow::{Context, Result};
+use camino::Utf8PathBuf;
+use clap::Parser;
+use log::{debug, info};
+use serde_json::{json, Value};
+use vscode::{BazelInfo, LaunchConfigGenerator};
+
+#[derive(Parser)]
+#[command(
+    name = "generate_launch_json",
+    about = "Generate VSCode launch.json configurations for debugging Bazel Rust targets"
+)]
+struct Args {
+    /// Space separated list of target patterns that comes after all other args.
+    #[clap(default_value = "@//...")]
+    targets: Vec<String>,
+
+    /// Output file for launch configurations. If relative, the path will be joined with `workspace_root`.
+    #[arg(short, long, default_value = ".vscode/launch.json")]
+    output: Utf8PathBuf,
+
+    /// Workspace root directory
+    #[arg(long, env = "BUILD_WORKSPACE_DIRECTORY")]
+    workspace_root: Option<Utf8PathBuf>,
+
+    /// Path to Bazel binary
+    #[arg(short, long, default_value = "bazel")]
+    bazel: Utf8PathBuf,
+
+    /// Only generate configurations (don't write files)
+    #[arg(long)]
+    dry_run: bool,
+
+    /// Don't pretty print generated JSON
+    #[arg(long)]
+    no_pretty: bool,
```

Comment:
Maybe `pretty` with default `true`, rather than default negation?

Reply:
Does clap support this?